Python/pytime.c

```diff
@@ -55,6 +55,14 @@
 #endif
 
 
+#ifdef MS_WINDOWS
```

Comment:
Yes, I think checking for `MS_WINDOWS` here is sufficient. `GetSystemTimePreciseAsFileTime()`, `QueryPerformanceCounter()`, and `QueryPerformanceFrequency()` are available in the system, app, and game partitions. The old use of `GetSystemTimeAdjustment()` actually wasn't supported for app builds. However, that function was used improperly anyway.